src/System.Collections.NonGeneric/ref/System.Collections.NonGeneric.cs

```diff
@@ -71,6 +71,15 @@ public partial class CaseInsensitiveComparer : System.Collections.IComparer
         public static System.Collections.CaseInsensitiveComparer DefaultInvariant { get { return default(System.Collections.CaseInsensitiveComparer); } }
         public int Compare(object a, object b) { return default(int); }
     }
+    [System.ObsoleteAttribute("Please use StringComparer instead.")] 
```

Comment:
> We decided to not add any new [Obsolete] attributes, but it looks like you're just bringing over ones that already exist.

Right, this is existing surface area marked as [Obsolete] in desktop.

> Incidentally I'm not sure what it would mean to add new obsoletes here on the contract/façade - presumably it wouldn't break anyone referencing mscorlib directly, and of course nobody referencing the contract if it's a newly added member.

There is one way I know of that this is a breaking change, though it's extremely corner case.  It's why this line got changed in the tests:
https://github.com/dotnet/corefx/pull/9967/files/f55add220f1f381ad56d393531bc5547101be80f#diff-760ec85d54d45aacc0b261c8423034ceR207
This line is passing null to both Hashtable arguments.  That results in an exception, but it compiles fine... because there's only one ctor that takes two reference type parameters.  This change adds another ctor that takes two reference type parameters, which makes passing null to both of them ambiguous to the compiler which you intended to target, causing compilation to fail until you disambiguate.